src/index.html

```diff
@@ -11,12 +11,112 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
```

Comment:
Lines of code have 80 chars max. Consider breaking this line for better readability.